frontend/src/components/common/header/index.css

```diff
@@ -48,99 +52,101 @@
   background-size: cover;
   background-position: center center;
   background-repeat: no-repeat;
-  position: absolute;
+  position: relative;
   top: 25px;
   left: 24px;
   z-index: 50;
 }
 
+.social-media {
+  color: #519b56;
+  width: 25px;
+  height: 25px;
+}
+
+#donate-button {
+  font-size: 20px;
+  font-weight: bold;
+  background-color: #fdd368;
+  border-radius: 7px;
+  border: none;
+  width: 140px;
+  height: 54px;
+  text-align: center;
+}
+
+.center-button {
+  margin-top: auto;
+  margin-bottom: auto;
+}
+
 @media all and (min-width: 768px) and (max-width: 2048px) {
   .ym-top {
-    background-color: #ffffff;
-    font-size: 24pt;
-    font-family: "GothamMedium";
     height: 140px;
-    width: 100%;
   }
-}
 
-@media all and (min-width: 320px) and (max-width: 425px) {
-  .ym-top-mobile {
-    background-color: #ffffff;
-    font-size: 12pt;
-    font-family: "GothamMedium";
+  .ym-bottom {
     height: 140px;
-    width: auto;
   }
 
-  #tsocial {
-    display: none;
+  .logo {
+    height: 150px;
+    width: 150px;
+  }
+}
+
+@media all and (min-width: 1440px) and (max-width: 2048px) {
+  .logo {
+    height: 180px;
+    width: 180px;
   }
 
   .nav-link-top {
-    padding: 0px;
+    margin-left: 100px;
+    margin-right: 100px;
   }
 
-  .logo {
-    display: none;
+  .nav-link {
+    margin-right: 15px;
   }
-}
 
-.ym-bottom {
-  background-color: #519b56;
-  font-size: 24pt;
-  font-family: "GothamMedium";
-  height: 140px;
-  color: white;
-  width: 100%;
+  .align-links {
+    padding-left: 180px;
+    color: white;
+  }
 }
 
-.donate-button {
-  font-size: 20px;
-  font-weight: bold;
-  background-color: #fdd368;
-  border-radius: 2px;
-  width: 140px;
-  height: 54px;
-  text-align: center;
-}
+@media all and (min-width: 320px) and (max-width: 768px) {
+  .ym-top {
+    font-size: 12pt;
```

Comment:
i think we want to be always using `px` instead of `pt` -> its the industry standard and they are not completely equal. In short it appears `pt` should only be used for on print and `px` should be used for screens 

I didn't know much about the difference but this SO has great info https://stackoverflow.com/questions/3557260/should-i-use-pt-or-px/27559564

Reply:
Converted pt to px in css file

Reply:
sweet looks great 👍